qaframework-bdd-tests/src/test/resources/features/refapp-2.x/13-appointment/appointment.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Appointment Management
+
+	Background:
+		Given a user clicks on Request Appointment link from the Patient dashboard
+		Then the system loads Request Appointment page
+		And a user fills in the request details
+		And a user clicks save button
+		Then the system adds appointment request into the appointment requests table
+		
+	@selenium
+	@appointment
+	Scenario: Create Appointment Block, Edit Appointment Block, Add Appointment and Delete Appointment Block
+		# User story: Create Appointment Block
+		When a user clicks Appointment scheduling app from the home page
+		Then the system loads the appointment scheduling page
+		And a user clicks on Manage Provider Schedules app
+		And the system loads Manage Provider Schedules page
+		And a user fills in the details of the appointment block
+		And a user clicks on the save button
+		Then the systems saves the appointment block
+		# User story: Edit Appointment Block
+		When a user clicks Appointment scheduling app from the home page
+		Then the system loads the appointment scheduling page
+		And a user clicks on the Manage Provider Schedules app
+		And the system loads Appointment Blocks page
+		And a user clicks on an appointment block
+		And a user clicks on Edit
+		And the system loads Edit Appointment Block section
+		And a user edits the Appointment block
+		And a user clicks on Save button
+		Then the systems saves the edited appointment block
+		# User story: Add Appointment
+		When a user clicks Appointment scheduling app from the home page
+		Then the system loads the appointment scheduling page
+		And a user clicks on Manage Appointments app
+		And a user searches for a patient
+		And a user clicks on the first patient
+		And the system loads Manage Appointments page
+		And a user enters service
```

Comment:
We can combine these aswell 
```
And a user enters service
And a user searches appointment
```
Into `And a user searches appointment`

Reply:
No objection at all.